dockers/MathJaxFonts/Dockerfile

```diff
@@ -23,39 +26,41 @@ RUN apt-get -qq update && apt-get -qqy install \
     woff-tools \
     pkg-config \
     libharfbuzz-dev \
-    libfreetype6-dev || true
-RUN gem install ttfunk --version 1.1.1
+    libfreetype6-dev \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/* \
+ && gem install ttfunk --version 1.1.1
 
 # Download yuicompressor
-RUN mkdir /usr/share/yui-compressor/
-RUN wget "https://github.com/yui/yuicompressor/releases/download/v2.4.8/yuicompressor-2.4.8.jar" -O /usr/share/yui-compressor/yui-compressor.jar
+ADD https://github.com/yui/yuicompressor/releases/download/v2.4.8/yuicompressor-2.4.8.jar /usr/share/yui-compressor/yui-compressor.jar
 
 # Download batik-ttf2svg.jar
-RUN wget "https://archive.apache.org/dist/xmlgraphics/batik/batik-1.7.zip"
-RUN unzip -qq batik-1.7.zip
-RUN mv batik-1.7/batik-ttf2svg.jar /usr/share/java/
+RUN wget "https://archive.apache.org/dist/xmlgraphics/batik/batik-1.7.zip" \
+ && unzip batik-*.zip batik-*/batik-ttf2svg.jar \
+ && mv batik-*/batik-ttf2svg.jar /usr/share/java/ \
+ && rm -r batik-*
```

Comment:
I'm curious about the benefit of combining multiple commands vs. running them separate?